src/test_report.py

```diff
@@ -121,11 +121,12 @@ def create_test_report(self, root_node, child_nodes):
 
     def run(self):
         sub_id = self._db.subscribe_node_channel(filters={
-            'op': 'completed',
+            'name': 'tarball',
```

Comment:
What about the completed checkout node?

Reply:
The script currently looks for child nodes, so it's including the results for all the tests that are child nodes of a tarball node.  We could listen to events about checkout nodes and then get the tarball node id from it, but at the moment there's always only one checkout and one child tarball node for each revision so it doesn't make any difference in practice (just more code if we listen for events related to checkout nodes).